kustomize/skipper/kustomization.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base/prometheus/
+  - ../base/flagger/
+  - https://raw.githubusercontent.com/zalando/skipper/master/docs/kubernetes/deploy/deployment/rbac.yaml
```

Comment:
We need to pin this to a fixed version instead of master.

Reply:
makes sense, adjusted